lib/core/repository/clumps/cms.rb

```diff
@@ -0,0 +1,25 @@
+module Core::Repository
+  module Clumps
+    class CMS < Core::Repository::Base
+      def initialize(options = {})
+        self.connection = Core::Registry::Connection[:cms]
+      end
+
+      def all
+        clumps_url = '/%{locale}/clumps.json' % { locale: I18n.locale }
```

Comment:
@tomas-stefano yes, we had this discussion the other day.

I was planning on coming back to it as a separate task to tidy these up as the CMS also needs changing (this one doesn't work under both, like some of the api end points).